README.md

```diff
@@ -8,34 +8,71 @@ This is studies project for Data Security course where project topic is blockcha
 
 Blockchain is a digital public ledger that records transactions. It is a distributed ledger that can be queried and verified. It is a public record of past transactions.
 
-### How blockchain works?
+### How blockchain works
 
 Blockchain works based on hash function and proof of work. Hash function is used to generate hash value and proof of work is used to verify hash value.
 
-### What is hash function?
+### What is hash function
 
-Hash function is used to generate hash value. Hash function is used to generate hash value.
+Hash function is used to generate hash value.
 
-### What is proof of work?
+### What is proof of work
 
-Proof of work is used to verify hash value. Proof of work is used to verify hash value.
+Proof of work is used to verify hash value
 
-### What is 51% attack?
+### What is 51% attack
 
 51% attack is a type of attack in which a computer program or a networked system is able to modify the data in a database or a file without the user's knowledge.
```

Comment:
What do you mean by "modify the data in a database or a file without the user's knowledge"?